packages/magento-payment-included/methods.tsx

```diff
@@ -1,8 +1,6 @@
-import {
-  PaymentMethodOptionsNoop,
-  PaymentMethodPlaceOrderNoop,
-  PaymentModule,
-} from '@graphcommerce/magento-cart-payment-method'
+import { PaymentModule } from '@graphcommerce/magento-cart-payment-method/Api/PaymentMethod'
+import { PaymentMethodOptionsNoop } from '@graphcommerce/magento-cart-payment-method/PaymentMethodOptionsNoop/PaymentMethodOptionsNoop'
+import { PaymentMethodPlaceOrderNoop } from '@graphcommerce/magento-cart-payment-method/PaymentMethodPlaceOrderNoop/PaymentMethodPlaceOrderNoop'
```

Comment:
I'm wondering if this also works:

```suggestion
import {
  PaymentMethodOptionsNoop,
  PaymentMethodPlaceOrderNoop,
  type PaymentModule,
} from '@graphcommerce/magento-cart-payment-method'
```